incubator/binding-pgsql-kafka/src/main/java/io/aklivity/zilla/runtime/binding/pgsql/kafka/internal/PgsqlKafkaConfiguration.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.binding.pgsql.kafka.internal;
+
+import io.aklivity.zilla.runtime.engine.Configuration;
+
+public class PgsqlKafkaConfiguration extends Configuration
+{
+    private static final ConfigurationDef PGSQL_KAFKA_CONFIG;
+
+    public static final IntPropertyDef KAFKA_TOPIC_REQUEST_TIMEOUT_MS;
+    public static final IntPropertyDef KAFKA_CREATE_TOPICS_PARTITION_COUNT;
+    public static final ShortPropertyDef KAFKA_CREATE_TOPICS_REPLICAS;
+    public static final PropertyDef KAFKA_AVRO_SCHEMA_NAMESPACE;
+
+    static
+    {
+        final ConfigurationDef config = new ConfigurationDef(String.format("zilla.binding.%s", PgsqlKafkaBinding.NAME));
+        KAFKA_TOPIC_REQUEST_TIMEOUT_MS = config.property("kafka.topic.request.timeout.ms", 30000);
+        KAFKA_CREATE_TOPICS_PARTITION_COUNT = config.property("kafka.create.topics.partition.count", 1);
+        KAFKA_CREATE_TOPICS_REPLICAS = config.property("kafka.create.topics.replicas", (short) 1);
+        KAFKA_AVRO_SCHEMA_NAMESPACE = config.property("kafka.avro.schema.namespace", "io.aklivity.zilla");
```

Comment:
This doesn't seem like the right value for the avro namespace as it would more naturally be part of the application, not part of zilla directly.

Seems like it should perhaps default to the pgsql `database` parameter instead, like `dev` and perhaps this can be a pattern that would substitute the pgsql `database` parameter into it, so we could configure it with something like `"com.example.{database}"` and have it resolve to `"com.example.dev"`. Then if there is no reference to `{database}` in the pattern, it would be used verbatim as it is now.

Reply:
Yeah, good suggestion. I was not sure what you recommend since com.example was too generic